lux/vis/VisList.py

```diff
@@ -272,7 +272,13 @@ def _ipython_display_(self):
         import luxwidget
 
         recJSON = LuxDataFrame.rec_to_JSON([recommendation])
-        self._widget = luxwidget.LuxWidget(currentVis={}, recommendations=recJSON, intent="", message="")
+        self._widget = luxwidget.LuxWidget(
+            currentVis={},
+            recommendations=recJSON,
+            intent="",
+            message="",
+            plottingScale=lux.config.plotting_scale,
```

Comment:
Based on the frontend changes, it looks like the only usage of `plottingScale` is to increase the size of the currentVis. Or is it also being used to determine the size of the widget divs? If the former is true, can we simply increase the currentVis size like what we did inside the AltairChart and MatplotlibChart before we send it to the frontend?
That way we can minimize adding a new traitlet variable, which requires the frontend and backend to be version-locked (i.e., not backwards-compatible) for the release.

Reply:
Actually, it's the latter. Plotting scale is mostly used in the frontend to determine the side of the widget divs, which requires a Traitlet variable.

Reply:
Hmm, if we really had to pass something to the frontend, maybe we go with something more generic so that we don't have to change the spec of the traitlet too much in the future. For example, we could called the traitlet variable `config` and it would be a dictionary like `{"plotting_scale":0.5, "some other config": "..."}`. That way if we want to pass other things in the future, we can pass it along in the config. This is similar to the approach that we took for #336, so that the [`all_column` flag is lumped together with the current vis spec](https://github.com/lux-org/lux/pull/336/files#diff-b3f5355f16af7450f6bf0669466ac57ae020ed8c0a56105e8ece9c8c035a8879L699), without the need to send an additional traitlet over. We do a similar thing with `vislib` parameter too to put it alongside each vis spec.

Reply:
Gotcha, I like the config variable idea and will include that in this PR! I agree that this is a future-proof way of more easily communicating with the frontend without additional traitlets in the future.